package.json

```diff
@@ -49,11 +50,15 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@jupyterlab/eslint-plugin-jinja": "^0.2.0",
```

Comment:
I removed @jupyterlab/eslint-plugin-jinja dependency because it was only used for the 2 files that are now ignored by eslint